packages/fast-breakpoint-tracker-react/src/breakpoints.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { Breakpoints, defaultBreakpoints, identifyBreakpoint } from "./breakpoints";
+
+describe("identifyBreakpoint", (): void => {
+    test("should return the correct breakpoint values", (): void => {
+        expect(identifyBreakpoint(0, defaultBreakpoints)).toBe(0);
+        expect(identifyBreakpoint(539, defaultBreakpoints)).toBe(0);
+        expect(identifyBreakpoint(540, defaultBreakpoints)).toBe(1);
+        expect(identifyBreakpoint(767, defaultBreakpoints)).toBe(1);
+        expect(identifyBreakpoint(768, defaultBreakpoints)).toBe(2);
+        expect(identifyBreakpoint(1083, defaultBreakpoints)).toBe(2);
+        expect(identifyBreakpoint(1084, defaultBreakpoints)).toBe(3);
+        expect(identifyBreakpoint(1399, defaultBreakpoints)).toBe(3);
+        expect(identifyBreakpoint(1400, defaultBreakpoints)).toBe(4);
+        expect(identifyBreakpoint(1778, defaultBreakpoints)).toBe(4);
+        expect(identifyBreakpoint(1779, defaultBreakpoints)).toBe(5);
```

Comment:
Retina displays go up quite a bit and 4k even more?  Should we extend to include at least up to 4k with a final 4001+?

Reply:
I mean, we can add tests but they won't validate anything here. Here we are testing that the method can correctly identify the correct breakpoint. If we want to add additional breakpoints to the defaultBreakpoints array I think that would be another task as I wouldn't want to add a breakpoint at random to cover larger screen sizes or densities.